src/Tempest/Database/Migrations/CreateMigrationsTable.php

```diff
@@ -4,22 +4,32 @@
 
 namespace Tempest\Database\Migrations;
 
+use Tempest\Database\DatabaseDriver;
 use Tempest\Database\Migration;
 use Tempest\Database\Query;
+use Tempest\Database\QueryStatement;
 
 final readonly class CreateMigrationsTable implements Migration
 {
+    public function __construct(
```

Comment:
You could add something like `$this->driver->createQueryStatement()`, so that you don't have to manually pass it in L26. I wouldn't bother with trying to be more clever than that for now.